python/sdk/setup.py

```diff
@@ -24,18 +24,18 @@
     "certifi>=2017.4.17",
     "python-dateutil>=2.1",
     "six>=1.10",
-    "mlflow>=1.2.0,<=1.23.0",
+    "mlflow>=1.2.0,<=1.23.0", #  for py3.11 due to proto -> "mlflow>=1.26.1",
     "google-cloud-storage>=1.19.0",
     "boto3>=1.9.84",
     "urllib3>=1.23",
     "PyPrind>=2.11.2",
-    "google-auth>=1.11.0,<2.0dev",
```

Comment:
Why is it removed?

Reply:
Yeah, agree with that statement. But try to limit the amount of change to only for supporting Python >3.7. This will make debugging issue easier compared to a big bang changes.